### DISCUSSION TITLE
modal对话框，需要对换确认、取消按钮的位置，我应该怎么做？

### DISCUSSION THREAD

**Discussion by wrgljylys** [ant-design/ant-design#44424] [Q&A]
antd 版本 5.4.0
对话框的确认按在右边、取消按钮在左边
现在需要做到 把确认按钮放左边、取消按钮放右边
我试了下在footer里重写，较麻烦且未能实现同样的布局样式

请问有没有好的实现方式？
谢谢。

**Comment by RedJue** [comment]
等这个pr  https://github.com/ant-design/ant-design/pull/44318
